### DISCUSSION TITLE
Cauchy Series Convergence Using Custom Output

### DISCUSSION THREAD

**Discussion by j-signorelli** [su2code/SU2#2049] [Q&A] [answered]
Hi,

I have written a custom output Stanton number for a simulation I am running, and would like to deem steady-state convergence as being reached when this number stops changing above a precision of 6.

I have defined in my config file the following:

    % Marker where TOTAL_HEATFLUX is calculated for
    MARKER_MONITORING = (wall)

    CUSTOM_OUTPUTS = 'STANTON_NUMBER : Function{TOTAL_HEATFLUX / (0.10* 100 *  1004.703 * 0.01 * (300 - 250))}'
    
    
    CONV_FIELD= STANTON_NUMBER
    CONV_CAUCHY_ELEMS=100 
    CONV_CAUCHY_EPS=1E-6

It does correctly say when I start up SU2 that `Convergence Field(s): STANTON_NUMBER`, however, when I add `CAUCHY` to screen outputs, nothing shows up. Doing `SU2_CFD -d config.cfg` also does not show `CAUCHY_STANTON_NUMBER` as an output option, but does show it for other non-custom outputs.

Is there something I am missing here? Is this feature of using custom outputs for convergence fields not available in SU2?

Thanks.

**Comment by pcarruscag** [comment]
This should be fixed in the develop branch, give it a try please.

**Reply by j-signorelli** [reply]
Thanks for your response - is there a particular commit you may please point me to that fixes this by any chance? I have some updates I've made that I need that uses the current release 7.5.1, and would like to just cherry-pick that commit into my own fork if possible. I tried searching for it but couldn't find it.

**Reply by pcarruscag** [reply, marked as answer]
https://github.com/su2code/SU2/pull/2047

**Reply by j-signorelli** [reply]
This worked, thank you!
